test/functional/framework.go

```diff
@@ -356,8 +356,8 @@ func (f *FluentdFunctionalFramework) WritesNApplicationLogsOfSize(numOfLogs, siz
 
 func (f *FluentdFunctionalFramework) WriteMessagesToLog(msg string, numOfLogs int, filename string) error {
 	logPath := filepath.Dir(filename)
-	cmd := fmt.Sprintf("bash -c 'mkdir -p %s;for n in {1..%d};do echo \"%s\" >> %s;done'", logPath, numOfLogs, msg, filename)
-	log.V(3).Info("Writing mesages to log with command", "cmd", cmd)
+	cmd := fmt.Sprintf("bash -c 'mkdir -p %s;for n in {1..%d};do echo \"%s\" >> %s;sleep 1s;done'", logPath, numOfLogs, msg, filename)
```

Comment:
What is the sleep waiting for? Sleeping 1s on every message written will slow down the tests quite a lot.

Reply:
@alanconway This is a workaround for an odd issue that doesn't seem to have anything to do with the security configuration of the fluentd container, yet it showed up on this branch.
Without this sleep I was getting a persistent failure of the 
```
test/functional/normalization/message_format_test.go
    It("should parse k8s audit log format correctly"
```
test, like so
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-logging-operator/1003/pull-ci-openshift-cluster-logging-operator-master-functional/1391588138463268864/build-log.txt 

log generator -> fluentd_1 -> fluentd_2 -> the code of the test

The test timed out waiting for the audit log to come out of fluentd_2. I figured that fluentd_1 doesn't pick up the log generator's log. I didn't dig deeper as soon as I came up with the workaround, it would be interesting to get to the bottom of this.

Reply:
@alanconway Here's the tech debt JIRA for this: https://issues.redhat.com/browse/LOG-1402